bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -16,6 +16,29 @@
 		</tags>
 	</channel-type>
 
+	<!-- Aux channel -->
+	<!--Only ON and OFF options are supported -->
+	<channel-type id="aux">
+		<item-type>String</item-type>
+		<label>Auxiliary</label>
+		<description>Controls an Auxiliary command (read/write)</description>
+		<state>
+			<options>
+				<option value="OFF">OFF</option>
+				<option value="ON">ON</option>
+				<option value="TOGGLE">TOGGLE</option>
+				<option value="STOP">STOP</option>
+				<option value="UP">UP</option>
+				<option value="DOWN">DOWN</option>
+				<option value="ENABLED">ENABLED</option>
+				<option value="DISABLED">DISABLED</option>
+				<option value="RESET_GEN">RESET_GEN</option>
+				<option value="RESET_BI">RESET_BI</option>
+				<option value="RESET_TRI">RESET_TRI</option>
+			</options>
```

Comment:
Why defining options if option label is always equal to option value ?
Generally, the option label is something understandable by a user and generally not all in uppercase. As an example, you could define "Enabled" for value "ENABLED".